src/codegeneration/InlineES6ModuleTransformer.js

```diff
@@ -0,0 +1,134 @@
+// Copyright 2015 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {CONST} from '../syntax/TokenType.js';
+import {ModuleTransformer} from './ModuleTransformer.js';
+import {
+  createBindingIdentifier,
+  createFunctionBody,
+  createImmediatelyInvokedFunctionExpression,
+  createScopedExpression,
+  createVariableStatement,
+  createUseStrictDirective
+  } from './ParseTreeFactory.js';
+import globalThis from './globalThis.js';
+import scopeContainsThis from './scopeContainsThis.js';
+import {IMPORT_SPECIFIER_SET} from '../syntax/trees/ParseTreeType.js';
+
+let anonInlineModules = 0;
+
+// transforms:
```

Comment:
Make this into a single comment.

Can you also provide some information what the usage of this is?